app/Http/Controllers/ConfidantController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Confidant;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+
+class ConfidantController extends Controller
+{
+    //Index page where all the confidants are shown for the user.
+    public function index()
+    {
+        return view('confidant.confidants', [
+            'confidants' => Confidant::all()
+        ]);
+
+    }
+
+    //This function refers to the admin page which will be changed but i just needed it for now :)
+    public function all(Confidant $confidant)
+    {
+        return view('admin.index', [
+            'confidant' => $confidant
+        ]);
+    }
+
+    //Show page where the selected confidants information is shown
+    public function show (Confidant $confidant)
+    {
+        return view('confidant.show', [
```

Comment:
Om variabelen mee te sturen naar een view, is het het makkelijkst om de 'compact' functie te gebruiken. Dat ziet er als volgt uit:
return view('usersView', compact('users', 'headTitle'));